argo-cd-apps/overlays/base/IAM/iam.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Remove this IAM folder and ESO one, only one copy in base/all-clusters is good enough

Reply:
Done.